Unix/tests/codec/mof/blue/test_lex.cpp

```diff
@@ -32,10 +32,20 @@
 
 #define TEST_ASSERT(x) NitsAssert(x, MI_T(""))
 
+#if (defined(sun) || defined(hpux) || defined(aix)) && defined(CONFIG_ENABLE_WCHAR)
+
+// 
+// This failes due to difficulty supporting four byte character sets in codec/mof/parser/utility.c
+// An issue has been filed.
+//
+#define DISABLE_WCHAR_TESTS 1
```

Comment:
I think I saw this in master already, it might create merge conflicts. Just a heads up.